lagunita/lms/templates/static_templates/copyright.html

```diff
@@ -73,13 +75,4 @@ <h2 id="copyright">${_('Copyright Complaints')}</h2>
     </section>
 </main>
 
-<script>
-    $(document).ready(function() {
-        var print_doc = '<input type="button" value="${_('Print This Document')}" class="print">';
-        $('#content section.copyright').prepend(print_doc).append(print_doc);
-        $('#content section.copyright input.print').click(function() {
-            window.print();
-            return false;
-        });
-    });
-</script>
+<%include file="print_tos.html" />
```

Comment:
also moved the print button, as that was all the same code repeated a bunch of times